### PR TITLE
Servlet task

### PR DESCRIPTION
**Pull request by romaanankin**

### CODE REVIEW

**Comment by boroda4436** [comment]
@romaanankin HW is not fully completed. Where are dao/service layers for login and registration?

File: src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -8,11 +8,40 @@
...
+    </servlet-mapping>
+    <servlet-mapping>
+
+        <servlet-name>main</servlet-name>
+        <url-pattern>/admin</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>main</servlet-name>
+        <url-pattern>/noAccess</url-pattern>
```
Comment (YatKat) [inline src/main/webapp/WEB-INF/web.xml]: Why do you have mapping for all pages? It's not convenient.

File: pom.xml [hunk 2]
```diff
@@ -79,6 +79,11 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
             <version>2.9.6</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-core</artifactId>
+            <version>5.1.3.RELEASE</version>
+        </dependency>
```
Comment (YatKat) [inline pom.xml]: As I understand we work without Spring and Security here.

Comment (romaanankin) [reply pom.xml]: have no idea how it appears in my project)

File: src/main/webapp/WEB-INF/web.xml (hunk 1 above)
Comment (romaanankin) [reply src/main/webapp/WEB-INF/web.xml]: for some reason its not work when you map /** , I just wanted to exclude "/servlet/" from path

**Pull request merged**